setup/create-wp-project/src/commands/theme.js

```diff
@@ -36,6 +36,15 @@ exports.handler = async (argv) => {
 	await clearConsole();
 	await writeIntro();
 
+	// Check if you are in the required folder.
+	const { path: requiredPath, isCorrectFolder } = await installPath('themes');
+
+	if (!isCorrectFolder) {
```

Comment:
The goal is to allow users to set up the project from the root directory, without being required to be inside the theme folder. The installation process will ensure that the theme is placed in the correct location.

So if they setup the project from the root, wp-content, wp-content/plugins no mater the setup should go to the root of the project where the WP-config is set add add the theme to the correct place.

The same thing should work with the plugin flag used for the plugin setup